script/docker-startup

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+rm -f tmp/pids/server.pid
+
+if [ "x${RAILS_SERVER}" = x"webrick" ] ; then
+  bundle exec rackup -s webrick -p "${BIND_PORT}" -o "${BIND_IP}"
+else
+  bundle exec rails s -p "${BIND_PORT}" -b "${BIND_IP}"
+fi
```

Comment:
I definitely like the idea of having an environment variable to make this change.

I've been following the pattern of the command is as close to the basic command for starting up the process as possible. Can `rackup` be used to start puma and then just use that?

Reply:
I second that. Currently playing around with the debugger setup and was just about to ask about something like that....